samples/nrf9160/aws_fota/Kconfig

```diff
@@ -10,80 +10,39 @@ config APP_VERSION
 	string "Application version"
 	default "v1.0.0"
 
-config USE_NRF_CLOUD
-	bool "Use nRF Cloud"
-	default n
-
-config CLOUD_CERT_SEC_TAG
-	depends on !USE_NRF_CLOUD
+config  CERT_SEC_TAG
 	int "Security tag for TLS credentials"
 	default 12345678
-
-config PROVISION_CERTIFICATES
-	depends on !USE_NRF_CLOUD
-	bool "Provision certificates from the certificates.h file"
-	select MODEM_KEY_MGMT
-	default n
 	help
-	  If enabled, the sample provisions server certificates into
-	  the modem by storing the certificates defined in the
-	  certificates.h file in the modem under the given security tag.
-	  Use this option only once to provision the device.
-	  The certificates are stored in the application binary and are
-	  therefore shown in the modem trace information. This is a
-	  security risk. After provisioning the certificates, turn off
-	  this option and compile and program the sample again.
-	  Also, do not share the binary that includes the compiled
-	  certificates with anyone.
+	  Security tag where the certificates to the AWS IoT MQTT broker is
+	  stored.
 
-config USE_CLOUD_CLIENT_ID
-	depends on !USE_NRF_CLOUD
+config USE_CUSTOM_CLIENT_ID
 	bool "Custom MQTT client ID"
-	default y
+	default n
```

Comment:
This line is not needed.

Reply:
I thought maybe defaulting to nrf-IMEI would be the best option as a default if you disagree I can change back the documentation to say the default is `your_client_id`

Reply:
I think that @simensrostad meant that `default n` is not needed as it's redundant (configs are disabled implicitly by default). IMO it's ok to use a nrf-IMEI as a default client ID since it's unique.

Reply:
Yes, That is correct.